parcels/scripts/get_examples.py

```diff
@@ -0,0 +1,110 @@
+"""Get example scripts, notebooks, and data files."""
+
+import argparse
+from datetime import datetime, timedelta
+import os
+import pkg_resources
+from progressbar import ProgressBar
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen
```

Comment:
That's a copy-and-paste artifact from the original `pull_data.py`.  I should clean it up.

> In principle we don't aim to be Python3-compliant in Parcel.

In the long run, this may become a problem as people are moving away from "Python 2": <http://www.python3statement.org/>. (IPython, for example, already stopped implementing new features in Python 2. Bug-fixes are still backported, though.)

Reply:
OK fine. I suggest to just leave it here then. At some point we do want to move to Python3, and having this here then helps conversion for sure!